### PR TITLE
added mobile styling and size adjustments

### PR DESCRIPTION
**Pull request by chris-w-mclean-shopify**
### WHY are these changes introduced?

- Added breakpoint to adjust the width of the container on small screens
- Adjusted desktop sizing to better match Figma doc
- Added a horizontal margin by default. I did it this way for two reasons. 1 the size where the container becomes flush with the browser screen doesn't match any Polaris breakpoints. 2. It keeps the horizontal margin styles in the same place as the vertical margin styles.

### How to 🎩

🖥 [Local development instructions](https://github.com/Shopify/polaris/blob/main/README.md#local-development)
🗒 [General tophatting guidelines](https://github.com/Shopify/polaris/blob/main/documentation/Tophatting.md)
📄 [Changelog guidelines](https://github.com/Shopify/polaris/blob/main/documentation/Versioning%20and%20changelog.md)

<details>
<summary>Copy-paste this code in <code>playground/Playground.tsx</code>:</summary>

```jsx
import React from 'react';
import {Page} from '../src';

export function Playground() {
  return (
    <Page title="Playground">
      {/* Add the code you want to test in here */}
    </Page>
  );
}
```

</details>

### 🎩 checklist

- [ ] Tested on [mobile](https://github.com/Shopify/polaris/blob/main/documentation/Tophatting.md#cross-browser-testing)
- [ ] Tested on [multiple browsers](https://help.shopify.com/en/manual/shopify-admin/supported-browsers)
- [ ] Tested for [accessibility](https://github.com/Shopify/polaris/blob/main/documentation/Accessibility%20testing.md)
- [ ] Updated the component's `README.md` with documentation changes
- [ ] [Tophatted documentation](https://github.com/Shopify/polaris/blob/main/documentation/Tophatting%20documentation.md) changes in the style guide
- [ ] For visual design changes, ping @ sarahill to update the Polaris UI kit

### CODE REVIEW

File: polaris-react/src/components/InContextLearning/InContextLearning.scss [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
   position: relative;
-  width: 250px;
+  width: 380px;
   border-radius: var(--p-border-radius-2);
   padding: var(--p-space-5);
   background-color: var(--p-surface);
   filter: drop-shadow(0px 0px 3px rgba(0, 0, 0, 0.1))
     drop-shadow(0px 4px 20px rgba(0, 0, 0, 0.15));
+
+  @include breakpoint-before(490px) {
```
Comment (chris-w-mclean-shopify) [inline polaris-react/src/components/InContextLearning/InContextLearning.scss:14]: I'm following convention from another Polaris component that is referencing this sizing directly. Both Tag and TextField do this.

File: polaris-react/src/components/InContextLearning/InContextLearning.tsx [hunk 2]
```diff
@@ -83,6 +84,7 @@ export function InContextLearning({onDismiss, title}: Props) {
               <div
                 className={styles.InContextLearning}
                 style={{
+                  ...horizontalMargin,
```
Comment (alololox) [inline polaris-react/src/components/InContextLearning/InContextLearning.tsx]: If you rebase you'll see these inline styles haved moved to the `scss` file, can we do similar here?

File: polaris-react/src/components/InContextLearning/InContextLearning.tsx [hunk 3]
```diff
@@ -71,6 +71,7 @@ export function InContextLearning({onDismiss, title}: Props) {
         activator={currentActivator}
         render={(state) => {
           const isAbove = state.positioning === 'above';
+          const horizontalMargin = {marginLeft: '.5rem', marginRight: '0.5rem'};
```
Comment (rockinrobin714) [inline polaris-react/src/components/InContextLearning/InContextLearning.tsx]: I moved this out in a different PR - if you pull in the latest you can move it into the InContextLearning.scss!

File: polaris-react/src/components/InContextLearning/InContextLearning.tsx (hunk 2 above)
Comment (rockinrobin714) [reply polaris-react/src/components/InContextLearning/InContextLearning.tsx]: GAH WHY DO YOU ALWAYS BEAT ME BY LIKE HALF A SECOND

File: polaris-react/src/components/InContextLearning/InContextLearning.scss [hunk 4]
```diff
@@ -1,11 +1,17 @@
+@import '../../styles/common';
+
 .InContextLearning {
   position: relative;
-  width: 250px;
+  width: 380px;
```
Comment (annmariedoyle-shopify) [inline polaris-react/src/components/InContextLearning/InContextLearning.scss:5]: Would we want to define these are variables? (ie desktop-width or something)

File: polaris-react/src/components/InContextLearning/InContextLearning.tsx (hunk 3 above)
Comment (chris-w-mclean-shopify) [reply polaris-react/src/components/InContextLearning/InContextLearning.tsx]: Taking care of it now.

File: polaris-react/src/components/InContextLearning/InContextLearning.tsx (hunk 2 above)
Comment (chris-w-mclean-shopify) [reply polaris-react/src/components/InContextLearning/InContextLearning.tsx]: Taking care of it now.

File: polaris-react/src/components/InContextLearning/InContextLearning.scss (hunk 4 above)
Comment (chris-w-mclean-shopify) [reply polaris-react/src/components/InContextLearning/InContextLearning.scss:5]: For a future iteration, sure, absolutely I think we should. There's so much fast change going on within these components at the moment though. I didn't want to enshrine those values to variables, have another PR leverage those, and then need to later undo those changes.

**Review by alololox — APPROVED** [review]

File: polaris-react/src/components/InContextLearning/InContextLearning.tsx (hunk 2 above)
Comment (chris-w-mclean-shopify) [reply polaris-react/src/components/InContextLearning/InContextLearning.tsx]: Done 👍

File: polaris-react/src/components/InContextLearning/InContextLearning.tsx (hunk 3 above)
Comment (chris-w-mclean-shopify) [reply polaris-react/src/components/InContextLearning/InContextLearning.tsx]: Done 👍
